src/lib.rs

```diff
@@ -86,23 +88,27 @@ impl Builder {
         self
     }
 
-    pub fn append_include(&mut self, include: String) -> &mut Self {
-        self.includes.push(include);
+    pub fn append_include(&mut self, include: impl Into<String>) -> &mut Self {
+        self.includes.push(include.into());
         self
     }
 
-    pub fn include_black_list(&mut self, include_black_list: Vec<String>) -> &mut Self {
-        self.include_black_list = include_black_list;
+    pub fn black_list(&mut self, black_list: Vec<String>) -> &mut Self {
```

Comment:
Better keep the same style on params with `files`.

Reply:
:+1: